modules/internal/ChapelAutoAggregation.chpl

```diff
@@ -87,9 +87,10 @@ module ChapelAutoAggregation {
     use CPtr;
     use AggregationPrimitives;
 
+    param defaultBuffSize = if CHPL_COMM == "ugni" then 4096 else 8096;
```

Comment:
Is it deliberately 8096 instead of 8192?

Reply:
Nope, good catch. I meant for it to be 8192. We don't actually require it to be a power of 2, so I'll probably leave it be for now. It's actually probably a good place to use the "good alloc size" API (https://github.com/chapel-lang/chapel/issues/5272) to size the buffers.

Reply:
Fixed in https://github.com/chapel-lang/chapel/pull/19380